packager/packager.js

```diff
@@ -192,17 +193,29 @@ function getDevToolsLauncher(options) {
       res.writeHead(200, {'Content-Type': 'text/html'});
       fs.createReadStream(debuggerPath).pipe(res);
     } else if (req.url === '/launch-chrome-devtools') {
+      if (webSocketProxy.isDebuggerConnected()) {
+        // Dev tools are already open; no need to open another session
+        res.end('OK');
+        return;
+      }
       var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
-      var script = 'launchChromeDevTools.applescript';
       console.log('Launching Dev Tools...');
-      execFile(path.join(__dirname, script), [debuggerURL], function(err, stdout, stderr) {
+      launcher(function(err, launch) {
         if (err) {
-          console.log('Failed to run ' + script, err);
+          console.error('Failed to initialize browser-launcher2', err);
+          next(err);
         }
-        console.log(stdout);
-        console.warn(stderr);
+        launch(debuggerURL, {
+          browser: 'chrome',
+          options: ['--disable-web-security'],
```

Comment:
That's fair. As an alternative, what do you think about adding a command line option to packager.js to disable SOP? Something like
```shell
node packager.js --disable-web-security
```